src/scripts/main.js

```diff
@@ -1 +1,88 @@
 'use strict';
+
+let promiseNumber;
+let firstCheck = false;
+let secondCheck = false;
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    promiseNumber = 1;
+    resolve();
+  });
+
+  setTimeout(() => {
+    reject();
+  }, 3000);
+});
+
+const promise2 = new Promise((resolve) => {
+  document.addEventListener('click', () => {
+    promiseNumber = 2;
+    resolve();
+  });
+
+  document.addEventListener('contextmenu', (eve) => {
+    eve.preventDefault();
+    promiseNumber = 2;
+    resolve();
+  });
+});
```

Comment:
The `promise2` promise resolves on both click and context menu events. Consider whether both events should trigger the same promise resolution, as this might lead to unexpected behavior if both events occur.